chroma-manager/tests/integration/existing_filesystem_configuration/utils/create_lustre_filesystem.py

```diff
@@ -203,73 +208,60 @@ def get_unused_device(self, server_name):
     def umount_devices(self, server_name):
         lustre_server = self.get_lustre_server_by_name(server_name)
         for device in lustre_server['device_paths']:
-            self.remote_command(
-                server_name,
-                "if mount | grep %s; then umount %s; fi;" % (device, device))
+            self.remote_command(server_name,
+                                "if mount | grep %s; then umount %s; fi;" %
```

Comment:
This is a (common mistake and) dangerous pattern match.  When looking for a string in a file, it is usually best to delimit the string if possible to avoid substring matches.  In the case of matching against `mount` (or `/proc/mounts`), a common false positive is to be `grep`ping for `/dev/sda` and to match on `/dev/sda1`.

In this case, you probably want to add a space delimiter before and after the `%s`, since you are changing this line of code anyway.

Reply:
This isn't new code, I just reformatted it.

Reply:
Right.  As before, if the code deserves reformatting, it at least deserves having obvious bugs fixed also.